### DISCUSSION TITLE
Building for Different Operating Systems

### DISCUSSION THREAD

**Discussion by SymmetricChaos** [emilk/egui#1145] [Q&A]
So I wrote a small application and sent the exe to a friend for commentary. However when they opened it on Windows 7 and Windows 8 the ui was weirdly scattered and stretched. I built the application in Windows 10. Do I have to target 7 or 8 to get it to run on those?

Here's what my friend saw:

![image](https://user-images.githubusercontent.com/42520289/150601817-420d4fa1-c19a-4324-9caa-652e2a7d7af0.png)

**Comment by emilk** [comment]
Seems similar to this issue: https://github.com/emilk/egui/issues/838

Are both the glium and glow backends broken?

Have your friends tried updating their graphics drivers?

**Reply by SymmetricChaos** [reply]
Its possible these systems are just too old to have appropriate drivers. I'll ask them to update and try it on a more recent computer.
